dev/com.ibm.ws.jsfContainer/bnd.bnd

```diff
@@ -52,5 +52,5 @@ instrument.disabled: true
 -testpath: \
     ../build.sharedResources/lib/junit/old/junit.jar;version=file, \
     com.sun.faces:jsf-impl;version=2.2.14, \
-    org.apache.myfaces.core:myfaces-impl;version=2.2.12, \
+    org.apache.myfaces.core:myfaces-impl;strategy=exact;version=2.2.12, \
```

Comment:
Was this added b/c we we're pulling in the 3.0.0 version without it and the test was failing?

Reply:
Asked in the Jakarta Channel. They said to use exact.
`EXACT - When this strategy is used, the version range must be a single version which is the version which is searched for. If multiple repositories contain the the exact version of the artifact, the artifact from the first repository is used.`